packages/remark-shaku-code-annotate/src/index.ts

```diff
@@ -227,6 +224,37 @@ function isCommentLine(line: IThemedToken[]) {
     )
   );
 }
+function traverseLines(lines: IThemedToken[][]) {
```

Comment:
why here it is renamed to `traverseLines`? the original `parseLines` is better I think

Reply:
okay let's change the name.